lib/ansible/module_utils/basic.py

```diff
@@ -2160,7 +2160,7 @@ def bytes_to_human(self, size):
     pretty_bytes = bytes_to_human
 
     def human_to_bytes(self, number, isbits=False):
```

Comment:
This method should be made static (along with the matching `bytes_to_human`), since `self` is not being used. That will simplify testing.